ansible/kayobe-automation-run-tempest.yml

```
@@ -6,8 +6,8 @@
     results_path_local: "{{ lookup('env', 'PWD') }}"
     rally_image: 'stackhpc/docker-rally'
     rally_tag: latest
-    rally_image_full: "{{ rally_image }}:{{ rally_tag }}"
-    rally_no_sensitive_log: true
+    rally_image_full: "{{ rally_docker_registry }}/{{ rally_image }}:{{ rally_tag }}"
+    rally_no_sensitive_log: false
```

```suggestion
    rally_no_sensitive_log: true
```
Lets keep the sensitive log bit as openrc creds can be leaked